### ISSUE TITLE
A11y_PowershellEditorServices_Installation_Screenreader:Narrator does not read the "Expand/Collapse" state of the buttons(Disable,Settings,Command,Language,Debugger,Color themes)

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61702]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10

**Additional Details:**
Screenreader ,MAS1.3.1

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Extension" button of "ActivityBar"->Enter "PowerShell" in "Search Extension" text field and click on "Install" button.
3.Start narrator & navigate using "Tab" key to  "Disable" button , and observe if narrator reads collapsed/Expanded for disable button.
4.Navigate to Settings, Commands, Languages, Debugger, Color themes under the Contributors Tab item , and observed that whether narrator reads the state of the buttons as collapsed/Expanded.

**Actual:**
Narrator does not read the Collapsed/Expanded state of the buttons(Disable, Settings, Commands, Language, Debugger, Color themes)

**Expected:**
Narrator should read the Collapsed/Expanded state of the button(Disable, Settings, Commands, Language, Debugger, Color themes)

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not able to know the state of the buttons as Collapsed/Expanded.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Does this issue occur when all extensions are disabled?: Yes
[A11y_PowershellEditorServices_Installation_Screenreader.pptx](https://github.com/Microsoft/vscode/files/2510164/A11y_PowershellEditorServices_Installation_Screenreader.pptx)

**Comment by isidorn** [comment]
Narratpr os mpt si[[prted
